clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -16,6 +16,7 @@
 #include "CGObjCRuntime.h"
 #include "CGOpenCLRuntime.h"
 #include "CGRecordLayout.h"
+#include "CGSYCLRuntime.h"
```

Comment:
Do you need this include here? I don't see you using internal aspects in this file.